src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Repository/LocationsDuckDbRepository.cs

```diff
@@ -45,12 +45,12 @@ await publicDataDbContext
         {
             using var appender = duckDbConnection.CreateAppender(table: LocationOptionsTable.TableName);
 
-            var insertRow = appender.CreateRow();
```

Comment:
Pretty surprised this was working before, wonder how that was?

Reply:
I know, I was surprised too 🙂 .

I think the explanation is in the implementation of `DuckDB.NET.Data.DuckDBAppenderRow` in 0.10.2 which back then looked like this:

https://github.com/Giorgi/DuckDB.NET/blob/71a2908767a58a48f39a7486683fc6350fe7a93e/DuckDB.NET.Data/DuckDBAppender.cs#L19-L31

and from the docs linked in that code comment we see this:

![image](https://github.com/user-attachments/assets/4fed947b-2919-453c-8f86-d28db8f1b1ae)

so it wasn't doing anything before.

In 0.10.3 `DuckDbAppender` was rewritten to not use native methods and the `CheckColumnAccess` method was added to the new `AppendValue` method by https://github.com/Giorgi/DuckDB.NET/commit/9c1b5c8d5ae7ffef3605b6e8fb0b0d69065caf06.